stacks/environment-aws/variables.tf

```diff
@@ -36,8 +36,8 @@ variable "dashboard_version" {
   default = "0.2.0-eb0f89d7cf8"
 }
 variable "artifactory_license" {}
-variable "bitbucket_token" {}
```

Comment:
delete these lines instead of commenting out

Reply:
Done